core/getting-started/unit-testing-with-fsharp-nunit/MathService.Tests/Tests.fs

```diff
@@ -5,14 +5,6 @@ open NUnit.Framework
 
 open MathService
 
-[<Test>]
```

Comment:
I understand why this existed, but with the NUnit changes it no longer makes sense so I completely removed it. Wanted to explicitly point it out as it looks like "no test, no problem".